### DISCUSSION TITLE
Original Release Campaign?

### DISCUSSION THREAD

**Discussion by Tormentor667** [cxong/cdogs-sdl#845] [Q&A] [answered]
First of all, brilliant work on this. I just stumbled upon the link here today and I am amazed - I played Cyberdogs back in the late 90s or early 2000s and wasn’t aware of this.

One question now though: If I want to play the original missions from back then, which campaign would I have to play? 

Thanks kindly in return, 
Kind regards 
Dan

**Comment by cxong** [comment]
The 5 original campaigns are included

**Comment by Tormentor667** [comment]
Thank you but what's their names?

**Comment by cxong** [comment]
They are the ones Ronny made

https://github.com/search?q=repo%3Acxong%2Fcdogs-sdl+%22Author%22%3A+%22Ronny+Wester%22&type=code

**Comment by Tormentor667** [comment]
Thanks kindly!

**Comment by Tormentor667** [comment]
Okay I checked these campaigns but what I actually meant are the very classic campaigns from the versy first Cyber Dogs release. Are they available?

**Comment by cxong** [comment]
No not yet, there is a very old and big ticket that you can follow #71 
Note that Cyberdogs is a different game from C-Dogs, they have different code bases. You can play them online here:
https://dosgames.com/game/cyberdogs
https://dosgames.com/game/c-dogs/

**Comment by Tormentor667** [comment]
Aw cool, so you actually reverse engineer Cyberdogs as campaigns for C-Dogs? I wasn’t aware that both are different games actually

**Comment by cxong** [comment, marked as answer]
Cyberdogs has been added as of version 2.0.0

**Comment by Tormentor667** [comment]
So the original campaigns are finally playable? :)
